Classes/Systems/NSAttributedStringSizing.swift

```diff
@@ -183,3 +183,44 @@ final class NSAttributedStringSizing: NSObject, ListDiffable {
     }
 
 }
+
+class LabelLayoutManager: NSLayoutManager {
+    
+    override func fillBackgroundRectArray(_ rectArray: UnsafePointer<CGRect>, count rectCount: Int, forCharacterRange charRange: NSRange, color: UIColor) {
+        
+        // Get the attributes for the backgroundColor attribute
+        var range = charRange
+        let attributes = textStorage?.attributes(at: charRange.location, effectiveRange: &range)
+        
+        // Ensure that this is one of our labels we're dealing with, ignore basic backgroundColor attributes
+        guard attributes?[MarkdownAttribute.label] != nil else {
+            super.fillBackgroundRectArray(rectArray, count: rectCount, forCharacterRange: charRange, color: color)
+            return
+        }
+        
+        // Define label rectangle and rounded path
+        let cornerRadius: CGFloat = 2
+        let rect = rectArray[0].insetBy(dx: -2, dy: 2)
+        let path = UIBezierPath(roundedRect: rect, cornerRadius: cornerRadius)
+        
+        // Define the CoreGraphics context
+        let context = UIGraphicsGetCurrentContext()
+        context?.setAllowsAntialiasing(true)
+        context?.setShouldAntialias(true)
+        context?.setLineWidth(cornerRadius * 2)
+        context?.setLineJoin(.round)
+        
+        // Draw the label background
+        context?.addPath(path.cgPath)
+        context?.drawPath(using: .fillStroke)
+        
+        // Add a small inset box-shadow to the label
+        UIColor.black.withAlphaComponent(0.2).set()
```

Comment:
Could we do a device-pixel-thick border and use `Styles.Colors.Gray.border` instead so its the same style [as this](https://github.com/rnystrom/GitHawk/blob/master/Classes/Labels/LabelTableCell.swift#L17-L19)?

Reply:
Opted to do the same styling as GitHub desktop where it's alpha based on the background so it's slightly darker rather than a fixed grey border.

Happy to make the change though

Reply:
Ya consistency with app > everything else